grafana-plugin/src/components/GTable/GTable.tsx

```diff
@@ -62,7 +62,7 @@ export const GTable = <RT extends DefaultRecordType = DefaultRecordType>(props:
             }
 
             expandable.onExpand && expandable.onExpand(newExpanded, record);
-            expandable.onExpandedRowsChange(newExpandedRowKeys);
+            expandable.onExpandedRowsChange && expandable.onExpandedRowsChange(newExpandedRowKeys);
```

Comment:
`expandable.onExpandedRowsChange?.(newExpandedRowKeys);` - same can be done for the line above